metatron-app/metatron-discovery
# fix mobile external login page

### Description

fix the font size of the pop-up layer on the mobile external login page

**Related Issue** : 



### How Has This Been Tested?

1. Go to external login page. (See PR #3298 )
2. Login the client.
3. Check the font size of the pop-up layer on the mobile external login page and the web page.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
